gamerun.cc

```diff
@@ -0,0 +1,47 @@
+#include "gamerun.h"
+using namespace std;
+
+/*
+TO DO:
+- Add pickMove() method to Human
+- Create Levelx subclasses for Computer, each with a pickMove() method
+*/
+
+int gameRun(Observer *white, Observer *black, ChessGame *gameBoard) {
+    int turn = gameBoard->getplayerTurn();
+    int toReturn = INT_MAX;
+    string comm = "";
+    string a = "";
+    string b = "";
+
+    while (toReturn == INT_MAX) {
+        if (turn == 1) {
+            cout << "White's Turn: ";
+            bool resignStatus = white->pickMove();       //so pickMove() will have bool return type
```

Comment:
is `pickMove()` just a function that gives the player the option to run the `move` and `resign` commands?

Reply:
Essentially, yes, in Human::pickMove(), a player works with the commands move/resign and if the player does resign, pickMove() returns 1.